src/views/router/router.module.scss

```diff
@@ -13,14 +13,13 @@ $nav-tab-height: 5rem;
 	width: 100vw;
 
 	display: flex;
-	flex-direction: column;
+	flex-direction: row;
 	justify-content: flex-start;
-	align-items: center;
+	align-items: stretch;
```

Comment:
Not sure about this change. Marking for review, but I might resolve this myself once I pull down the code and check it out.

Reply:
Okay, pulled it down and understand the change.

We can just remove the property instead of setting it to stretch (stretch is default behavior here, I believe).
```suggestion
```